### ISSUE TITLE
Usage of the model mentioned in the paper?

### ISSUE THREAD

**Issue by emilpaulitz** [AlexanderKroll/SPOT#1]
Thank you for developing this model and publishing it!

I used the web server for a preliminary test and would like to use the model now for a greater number of transporter-substrate pairs. Therefore, I would like to install the Python version locally. 

I think I created the conda environment from the readme how I am supposed to. Now, how can I obtain the model that is trained on all train and test data? (From your paper: "We also developed a Python function that is available at https://github.com/AlexanderKroll/SPOT. For these prediction models, we trained gradient boosted decision tree models on both the training and test sets to expand the amount of training data, and thus improve the model’s performance.")

I tried running the jupyter notebook "model_fitting/A2 - Training Transporter-Substrate-Prediction model.ipynb", but was met with an XGBoost error: XGBoost version not compiled with GPU support. It seems it would take some time to resolve this. Is this even the correct way? Would it be possible to share the model you have already trained so I might get around solving this issue as I do not want to re-train the model anyway?

Any help would be greatly appreciated.
